EIPS/eip-7432.md

```diff
@@ -282,15 +274,18 @@ for complex tuple types.
 
 * Compliant contracts MUST implement the `IERC7432` interface.
 * A role is represented by a `bytes32`, and it's RECOMMENDED to use the `keccak256` of the role's name for this purpose:
-`bytes32 role = keccak256("ROLE_NAME")`.
+  `bytes32 role = keccak256("ROLE_NAME")`.
 * `grantRole` function MUST revert if the `_expirationDate` is in the past, and MAY be implemented as `public` or
-`external`.
+  `external`.
 * `revokeRole` function MAY be implemented as `public` or `external`.
-* The `hasRole` function MAY be implemented as `pure` or `view`.
-* The `hasRole` function SHOULD return `false` if `_supportsMultipleAssignments` is `false` and last role assignment is
-not to `_grantee` (see [Unique and Non-Unique Roles](#unique-and-non-unique-roles) for more).
-* The `roleData` function MAY be implemented as `pure` or `view`.
-* The `roleData` function SHOULD return `""` if the role assignment does not exist.
+* The `hasRole` function MAY be implemented as `pure` or `view` and SHOULD only confirm if the role assignment exists
+  and is not expired (supports simultaneous role assignments).
+* The `hasUniqueRole` function MAY be implemented as `pure` or `view` and SHOULD check if the assignment exists, is not
+  expired and is the last one granted (does not support simultaneous role assignments).
+* The `roleData` function MAY be implemented as `pure` or `view`, and SHOULD return an empty array of bytes if the role
+  assignment does not exist.
+* The `roleExpirationDate` function MAY be implemented as `pure` or `view`, and SHOULD return `uint64(0)` if the role
```

Comment:
The function signature in interface already implies that will be returned a uint64

```suggestion
* The `roleExpirationDate` function MAY be implemented as `pure` or `view`, and SHOULD return zero if the role
```